docs/source/install.rst

```diff
@@ -1,89 +1,99 @@
-Install Guide
-=============
+Installation Guide
+==================
 
-.. _before_install:
+Recommended Environments
+------------------------
 
-Before installing Chainer
--------------------------
+We recommend these Linux distributions.
 
-We recommend these platforms.
-
-* `Ubuntu <http://www.ubuntu.com/>`_ 14.04 LTS 64bit
+* `Ubuntu <http://www.ubuntu.com/>`_ 14.04/16.04 LTS 64bit
 * `CentOS <https://www.centos.org/>`_ 7 64bit
 
-Chainer is supported on Python 2.7.6+, 3.4.3+, 3.5.1+.
-Chainer uses C++ compiler such as g++.
-You need to install it before installing Chainer.
-This is typical installation method for each platform::
+The following versions of Python can be used: 2.7.6+, 3.4.3+, 3.5.1+, and 3.6.0+.
 
-  # Ubuntu 14.04
-  $ apt-get install g++
+.. note::
 
-  # CentOS 7
-  $ yum install gcc-c++
+   We are testing Chainer automatically with Jenkins, where all the above *recommended* environments are tested.
+   We cannot guarantee that Chainer works on other environments including Windows and macOS (especially with CUDA support), even if Chainer looks running correctly.
 
-If you use old ``setuptools``, upgrade it::
 
-  $ pip install -U setuptools
 
+Dependencies
+------------
 
-Install Chainer
----------------
+Before installing Chainer, we recommend to upgrade ``setuptools`` if you are using an old one::
 
-Chainer depends on these Python packages:
+  $ pip install -U setuptools
 
-* `NumPy <http://www.numpy.org/>`_ 1.9, 1.10, 1.11
-* `Six <https://pythonhosted.org/six/>`_ 1.9
+The following Python packages are required to install Chainer.
+The latest version of each package will automatically be installed if missing.
 
-Caffe model support
+* `NumPy <http://www.numpy.org/>`_ 1.9, 1.10, 1.11, 1.12
+* `Six <https://pythonhosted.org/six/>`_ 1.9+
 
-* `Protocol Buffers <https://developers.google.com/protocol-buffers/>`_
-* protobuf>=3.0.0 is required for Py3
+The following packages are optional dependencies.
+Chainer can be installed without them, in which case the corresponding features are not available.
 
-All of the above libraries are automatically installed with ``pip`` or ``setup.py``.
+* CUDA/cuDNN support
 
-CUDA/cuDNN support
+  * `cupy <http://docs.cupy.chainer.org/>`_
```

Comment:
I though it is not needed now because Chainer 2.0.0 will support any version of CuPy, but now I think it is better to clarify the version. I will fix it.